core/src/components/toast/toast.tsx

```diff
@@ -691,11 +719,11 @@ export class Toast implements ComponentInterface, OverlayInterface {
   }
 
   render() {
-    const { layout, el, revealContentToScreenReader, header, message } = this;
+    const { layout, el, revealContentToScreenReader, theme, header, message } = this;
     const allButtons = this.getButtons();
     const startButtons = allButtons.filter((b) => b.side === 'start');
     const endButtons = allButtons.filter((b) => b.side !== 'start');
-    const theme = getIonTheme(this);
+    const shape = this.getShape();
```

Comment:
Read comment above regarding theme.

```suggestion
    const theme = getIonTheme(this);
    const shape = this.getShape();
```